indico/modules/events/surveys/templates/emails/new_submission_email.txt

```diff
@@ -0,0 +1,11 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    {% trans %}New survey submission{% endtrans %}
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    New submission has been made to the survey {{ submission.survey.title }}, by {{ 'anonymous user' if not submission.user else submission.user.full_name -}}
```

Comment:
Missing "a" at the beginning of the sentence it and sounds strange nonetheless. I'd use different sentences for anon/non-anon.